fix page share guide according to notion's new doc (#188)
ramnes/notion-sdk-py: examples/first_project/README.md

Lastly we need a notion page to experiment with.
Duplicate [this page](https://www.notion.so/notion-sdk-py-540f8e2b79914654ba103c5d8a03e10e)
in your workspace.

> After creating an integration in [Notion](https://www.notion.com/my-integrations), [share](https://developers.notion.com/docs/authorization) the page with the integration. To do so, visit the page in your Notion workspace, click the ••• menu at the top right of a page, scroll down to `Add connections`, and use the search bar to find and select the integration from the dropdown list.

Reviewer: ```suggestion
After [creating an integration](https://www.notion.com/my-integrations),
[share](https://developers.notion.com/docs/authorization) the page with the
integration. To do so, visit the page in your Notion workspace, click the •••
menu at the top right of a page, scroll down to `Add connections`, and use the
search bar to find and select the integration from the dropdown list.
```

> After creating an integration in [notion site](https://www.notion.com/my-integrations), you should manually [shared](https://developers.notion.com/docs/authorization) the page with the integration. To share a page with an integration, visit the page in your Notion workspace, click the ••• menu at the top right of a page, scroll down to Add connections, and use the search bar to find and select the integration from the dropdown list.

Reviewer: ```suggestion
After creating an integration in [Notion](https://www.notion.com/my-integrations), [share](https://developers.notion.com/docs/authorization) the page with the integration. To do so, visit the page in your Notion workspace, click the ••• menu at the top right of a page, scroll down to `Add connections`, and use the search bar to find and select the integration from the dropdown list.
```
